corehq/apps/app_manager/tests/test_case_list_form.py

```diff
@@ -404,7 +404,7 @@ def test_target_module_different_datums(self):
             './entry')
 
     def test_case_list_form_requires_parent_case_but_target_doesnt(self):
-        factory = AppFactory(build_version='2.9.0/latest')
```

Comment:
Do you know what this `/latest` thing is?  It sounds like "the latest build, but at least 2.9.0" Could this exist in the real world too?

Reply:
No seeing an instance when version is set with `latest` in it on [HQ](https://www.commcarehq.org/builds/edit_menu/). 
Probably just a quick way of setting the version and the latest flag. I don't see how its set from the HQ interface though.

Reply:
It might not come from the UI.  Here's a bit of code that sets that version format:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L4837-L4837
I'm not sure what it does - just found that from grepping around.  There's also this constant that may factor in somewhere:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L191-L191
Anyways, I don't know how these are used, but changing the code to not accept that format seems risky.

@orangejenny Do you know anything about this version format?

Reply:
Looking at the first link there is a comment there
> # scrape for old conventions and get rid of them

Looks like this was used earlier and then there was this set up to migrate that from old convention where it was stored on Application to new convention where its now on BuildSpec?

also pinging @sravfeyn for hints.. looks like he did some work on this in https://github.com/dimagi/commcare-hq/pull/17614/files

Reply:
The word 'latest' in https://github.com/dimagi/commcare-hq/pull/17614/ is not really a string that would be appended to a build version (so, not related to this issue). It's just an HTML form value to keep track of a certain setting of the app.

Reply:
@esoergel 
after some digging found this 
https://github.com/dimagi/commcare-hq/commit/bd56ca7d3aea35925013e458c710e68f8139437d
blast from the past.

Reply:
If you look at the code here for [BuildSpec.from_string](https://github.com/dimagi/commcare-hq/blob/049b1f45fcdcb88d79d4804728055f514a51dfa9/corehq/apps/builds/models.py#L208) you can see that `latest` means the latest build for this version.

Looking at the [full list of versions of CommCare](https://www.commcarehq.org/builds/) you'll notice that there are multiple builds with the same version number.

So we can't just remove `latest`.

Reply:
Offlined about this but updating here as well. `AppFactor` passes the version directly into the build spec so the `/latest` part doesn't get parsed. So removing it from these places makes sense.